### PR TITLE
#903  Create slider for mobile version

### LINKED ISSUE #903: main.less:449: Markup mobile tile-about and text for main...
The puzzle `891-9180fe3b` from #891 has to be resolved: 

https://github.com/fidals/shopelectro/blob/bc1d3db1da1faa557d353c2ff9ac4dbfe9296c5c/front/less/pages/main.less#L449-L449

The puzzle was created by @duker33 on 26-Jun-19. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by duker33**
Closes #903

image

### CODE REVIEW

File: front/js/components/main.es6 [hunk 1]
```diff
@@ -8,12 +8,23 @@
     $timeTag: $('.js-select-time'),
   };
 
+  // @todo #903:30m  Move SCREENS const to configs.es6 file.
```
Comment (artemiy312) [inline front/js/components/main.es6]:
```suggestion
  // @todo #903:15m  Move SCREENS const to configs.es6 file.
```

It is only about moving the single variable

File: front/js/components/main.es6 [hunk 2]
```diff
@@ -38,6 +49,16 @@
     );
   }
 
+  function initSlider() {
+    if ($(window).width() < SCREENS.sm) {
+      $('.tile-about .row').slick({
```
Comment (artemiy312) [inline front/js/components/main.es6]: We should check what will do the library on window resizing and probably make a handler for transitions between sizes (e.g. `sm -> lg`)

**Comment by artemiy312** [comment]
Let's make a test for it

File: front/js/components/main.es6 (hunk 2 above)
Comment (duker33) [reply front/js/components/main.es6]: i'll subtask it

**Pull request merged**